src/test/kotlin/com/zepben/evolve/cim/iec61970/base/wires/SynchronousMachineTest.kt

```diff
@@ -0,0 +1,137 @@
+    internal fun accessorCoverage() {
+        val synchronousMachine = SynchronousMachine()
+
+        assertThat(synchronousMachine.baseQ, equalTo(null))
+        assertThat(synchronousMachine.condenserP, equalTo(null))
+        assertThat(synchronousMachine.earthing, equalTo(false))
+        assertThat(synchronousMachine.earthingStarPointR, equalTo(null))
+        assertThat(synchronousMachine.earthingStarPointX, equalTo(null))
+        assertThat(synchronousMachine.ikk, equalTo(null))
+        assertThat(synchronousMachine.maxQ, equalTo(null))
+        assertThat(synchronousMachine.maxU, equalTo(null))
+        assertThat(synchronousMachine.minQ, equalTo(null))
+        assertThat(synchronousMachine.minU, equalTo(null))
+        assertThat(synchronousMachine.mu, equalTo(null))
+        assertThat(synchronousMachine.r, equalTo(null))
+        assertThat(synchronousMachine.r0, equalTo(null))
+        assertThat(synchronousMachine.r2, equalTo(null))
+        assertThat(synchronousMachine.satDirectSubtransX, equalTo(null))
+        assertThat(synchronousMachine.satDirectSyncX, equalTo(null))
+        assertThat(synchronousMachine.satDirectTransX, equalTo(null))
+        assertThat(synchronousMachine.x0, equalTo(null))
+        assertThat(synchronousMachine.x2, equalTo(null))
+        assertThat(synchronousMachine.type, equalTo(SynchronousMachineKind.UNKNOWN))
+        assertThat(synchronousMachine.operatingMode, equalTo(SynchronousMachineKind.UNKNOWN))
+
+        synchronousMachine.fillFields(NetworkService())
+
+        assertThat(synchronousMachine.baseQ, equalTo(1.1))
+        assertThat(synchronousMachine.condenserP, equalTo(2))
+        assertThat(synchronousMachine.earthing, equalTo(true))
+        assertThat(synchronousMachine.earthingStarPointR, equalTo(3.3))
+        assertThat(synchronousMachine.earthingStarPointX, equalTo(4.4))
+        assertThat(synchronousMachine.ikk, equalTo(5.5))
+        assertThat(synchronousMachine.maxQ, equalTo(6.6))
+        assertThat(synchronousMachine.maxU, equalTo(7))
+        assertThat(synchronousMachine.minQ, equalTo(8.8))
+        assertThat(synchronousMachine.minU, equalTo(9))
+        assertThat(synchronousMachine.mu, equalTo(10.10))
+        assertThat(synchronousMachine.r, equalTo(11.11))
+        assertThat(synchronousMachine.r0, equalTo(12.12))
+        assertThat(synchronousMachine.r2, equalTo(13.13))
+        assertThat(synchronousMachine.satDirectSubtransX, equalTo(14.14))
+        assertThat(synchronousMachine.satDirectSyncX, equalTo(15.15))
+        assertThat(synchronousMachine.satDirectTransX, equalTo(16.16))
+        assertThat(synchronousMachine.x0, equalTo(17.17))
+        assertThat(synchronousMachine.x2, equalTo(18.18))
+        assertThat(synchronousMachine.type, equalTo(SynchronousMachineKind.generatorOrMotor))
+        assertThat(synchronousMachine.operatingMode, equalTo(SynchronousMachineKind.generator))
+    }
+
+    @Test
+    internal fun `add reactiveCapabilityCurve`() {
```

Comment:
todo: Again, please replace this test and the others with a test using `PrivateCollectionValidator`

Reply:
done